plugins/router/f5/f5.go

```diff
@@ -0,0 +1,555 @@
+package f5
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/golang/glog"
+)
+
+type F5Error struct {
+	verb           string
+	url            string
+	httpStatusCode int
+	err            error
+	JsonStatusCode int     `json:"code"`
```

Comment:
Why the json definition on these two fields.  Also, I didn't see this field being used anywhere